src/test/scala/cromwell/binding/ThreeStepImportNamespaceSpec.scala

```diff
@@ -0,0 +1,110 @@
+package cromwell.binding
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class ThreeStepImportNamespaceSpec extends FlatSpec with Matchers {
+  val psTaskWdl = """
+    |task ps {
+    |  command {
+    |    ps
+    |  }
+    |  output {
+    |    File procs = stdout()
+    |  }
+    |}""".stripMargin
+
+  val cgrepTaskWdl = """
+    |task cgrep {
+    |  command {
+    |    grep '${pattern}' ${File in_file} | wc -l
+    |  }
+    |  output {
+    |    Int count = read_int(stdout())
+    |  }
+    |}""".stripMargin
+
+  val wcTaskWdl = """
+    |task wc {
+    |  command {
+    |    wc -l ${File in_file}
+    |  }
+    |  output {
+    |    Int count = read_int(stdout())
+    |  }
+    |}""".stripMargin
+
+  val workflowWdl = """
+    |import "ps" as ns1
+    |import "cgrep" as ns2
+    |import "wc" as ns3
+    |
+    |workflow three_step {
+    |  call ns1.ps
+    |  call ns2.cgrep {
+    |    input: in_file=ns1.ps.procs
+    |  }
+    |  call ns3.wc {
+    |    input: in_file=ns1.ps.procs
+    |  }
+    |}""".stripMargin
+
+  def resolver(importUri: String): WdlSource = {
+    importUri match {
+      case "ps" => psTaskWdl
+      case "cgrep" => cgrepTaskWdl
+      case "wc" => wcTaskWdl
+      case _ => throw new RuntimeException(s"Can't resolve $importUri")
+    }
+  }
+
+  val binding = WdlBinding.process(workflowWdl, resolver _)
+
+  "WDL file with imports" should "Have 1 executable (1 workflow)" in {
+    binding.executables.size shouldEqual 1
```

Comment:
This doesn't need to be updated, but 

`binding.executables should have size 1` 

or 

`binding.tasks shouldBe empty`

would probably be more readable/idiomatic.